README.md

```diff
@@ -55,7 +55,7 @@ jobs:
       with:
           # Nix Flakes doesn't work on shallow clones
           fetch-depth: 0
```

Comment:
This is not true anymore, btw

Reply:
addressed.

Reply:
I'm not sure if that will work with the current installer

Reply:
@Kha do you have a project you can test this on?

Reply:
Ah sorry, I didn't check with the referenced nix-flakes-installer release. My setup is a bit customized as described in #63, but it does work with the (very recent) Nix master tarball there. AFAICT, support for shallow clones was implemented in https://github.com/NixOS/nix/commit/7d554f295ca0a8947042f367adb4f198730696be, so perhaps the nix-flake-installer release should simply be bumped, which sounds like a good idea anyway. And it would be great to test the flake example in this repo's CI.

Reply:
That's waiting on https://github.com/cachix/install-nix-action/pull/57 which is blocked on https://github.com/numtide/nix-flakes-installer which is waiting on https://github.com/NixOS/nix/issues/4047

Reply:
left out for now.